modules/sandboxed-containers-metrics-list.adoc

```diff
@@ -0,0 +1,16 @@
+// Module is included in the following assemblies:
+//
+// * sandboxed_containers/deploying_sandboxed_containers.adoc
+
+[id="sandboxed-containers-metrics-list_{context}"]
+= About {sandboxed-containers-first} metrics
+
+{sandboxed-containers-first} metrics enable administrators to monitor how their sandboxed containers are running.
+
+{sandboxed-containers-first} metrics are collected for the following categories:
+
+Kata agent metrics:: Kata agent metrics display information about the kata agent process running in your sandboxed containers. These metrics include data from `/proc/<pid>/[io, stat, status]`.
+Kata guest OS metrics:: Kata guest OS metrics display data from the gues OS running in your sandboxed containers. These metrics include data from `/proc/[stats, diskstats, meminfo, vmstats]` and `/proc/net/dev`.
+Hypervisor metrics:: Hypervisor metrics display data regarding the hypervisor process running the sandboxed containers. These metrics mainly include data from `/proc/<pid>/[io, stat, status]`.
+Kata monitor metrics:: Kata monitor metrics display detailed information about the resource usage of the kata-monitor process itself. These metrics also include counters from Prometheus data collection.
```

Comment:
I'm not sure whether "kata monitor" is documented anywhere else.
If it's not, maybe we need to explain that "kata monitor" is the process that gathers the metrics data and make it available to Prometheus?
This can be inferred from the last sentence, but as we don't link to the upstream doc (with architecture), it may not be obvious.

Reply:
@littlejawa  How's this?

Kata monitor is the process that gathers metric data and makes it available to Prometheus. The kata monitor metrics display detailed information about the resource usage of the kata-monitor process itself. These metrics also include counters from Prometheus data collection.

Reply:
Looks good to me :) Thanks